### PR TITLE
re-create pr https://github.com/imsnif/synp/pull/32

### PR DESCRIPTION
**Pull request by bluelovers**
re-create pr before rename pkg name

https://github.com/imsnif/synp/pull/32

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+      dependencies
+    })
+  },
+  npmToYarnCore (packageLockFileString, packageDir) {
+    const packageLock = JSON.parse(packageLockFileString)
+    const nodeModulesTree = nmtree(packageDir)
+    const yarnTree = buildYarnTree(nodeModulesTree, packageLock)
+    return lockfile.stringify(yarnTree)
+  }
+}
```
Comment (imsnif) [inline lib/index.js]: I like this change, but I don't like the function names. Could we change them to: `convertYarnToNpm` and `convertNpmToYarn`? What do you think?

Comment (bluelovers) [reply lib/index.js]: hmm? the name is from ur
```ts
const { buildYarnTree, buildNpmTree } = require('./tree')
```

or u mean this 2

```
yarnToNpmCore
npmToYarnCore
```

Comment (imsnif) [reply lib/index.js]: Yeah, I don't like the -core suffix.

**Comment by imsnif** [comment]
Merged and published as 1.5.0. Thanks for this, @bluelovers !

**Pull request merged**